restapi/error.go

```diff
@@ -123,6 +128,9 @@ func prepareError(err ...error) *models.Error {
 		if len(err) > 2 && err[2] != nil {
 			log.Print("debugging error: ", err[2].Error())
 		}
+		if err[0].Error() == "Specified remote tier already exists" {
```

Comment:
This can be changed to something like this:

```suggestion
                 errRemoteTierExists := errors.New("Specified remote tier already exists")
                 if errors.Is(err[0], errRemoteTierExists) {
```

Reply:
done